### DISCUSSION TITLE
how to close tabs/pages?

### DISCUSSION THREAD

**Discussion by deependujha** [LazyVim/LazyVim#5191] [Q&A] [answered]
I'm a newbie and using lazynvim starter.

<img width="1434" alt="Image" src="https://github.com/user-attachments/assets/628feff7-acdf-4408-a029-aa9a9d1b452b" />

As you can see, multiple pages/tabs are open (readme, hello.py, etc).

1. How can I switch between them?
2. How can I close a specific page/tab?
3. What is the term used for them (page, tab, or something else)?

**Comment by folke** [comment, marked as answer]
those are buffers, not tabs
close them with `<leader>bd` (buf delete)
switch with `<s-h>` and `<s-l>`.
Check the lazyvim website for all buffer related keymaps.
